src/test/components/formComponents/sourceWizardSummary.test.js

```diff
@@ -179,6 +179,41 @@ describe('SourceWizardSummary component', () => {
       ).toBeInTheDocument();
     });
 
+    it('Oracle Cloud Infrastructure - OCID', () => {
+      formOptions = {
+        getState: () => ({
+          values: {
+            source: { name: 'source_name' },
+            application: {
+              application_type_id: '2',
+              extra: {
+                bucket: 'cost-management',
+                bucket_namespace: 'yyhddfgeeu',
+                bucket_region: 'uk-london-1',
+                compartment_id: 'compartment-id',
+                policy_compartment: 'policy-compartment',
+              },
+            },
+            source_type: 'oracle-cloud-infrastructure',
+            authentication: { username: 'ocid.uuid', authtype: 'ocid' },
+          },
+        }),
+      };
+
+      const { container } = render(<SourceWizardSummary {...initialProps} formOptions={formOptions} />);
+
+      const data = getListData(container);
+
+      expect(data).toEqual([
+        ['Name', 'source_name'],
+        ['Source type', 'Oracle Cloud Infrastructure'],
+        ['Application', 'Cost Management'],
+        ['Bucket', 'cost-management'],
+        ['Bucket namespace', 'yyhddfgeeu'],
```

Comment:
Can we use a slightly more descriptive value for the text?

Reply:
I used the same they are using in the mocks.

Reply:
Fixed